deploy/test-environments/fleet_api/src/api/common_api.py

```diff
@@ -249,9 +251,45 @@ def get_cloud_security_posture_version(cfg: Munch, prerelease: bool = True) -> s
         return None
 
 
-def update_package_version(cfg: Munch, package_version: str):
+def get_cloud_security_posture_version(cfg: Munch, prerelease: bool = True) -> str:
+    """
+    Retrieve the version of the cloud_security_posture package.
+
+    Args:
+        cfg (Munch): Configuration object containing Kibana URL, authentication details, etc.
+        prerelease (bool, optional): Flag indicating whether to include
+                                        prerelease versions.Defaults to True.
+
+    Returns:
+        str: The version of the cloud_security_posture package, or None if the API call fails.
+
+    """
+    return get_package_version(
+        cfg=cfg,
+        package_name="cloud_security_posture",
+        prerelease=prerelease,
+    )
+
+
+def get_cloud_defend_version(cfg: Munch, prerelease: bool = True) -> str:
+    """
+    Retrieve the version of the cloud_defend package.
+
+    Args:
+        cfg (Munch): Configuration object containing Kibana URL, authentication details, etc.
+        prerelease (bool, optional): Flag indicating whether to include
+                                        prerelease versions.Defaults to True.
+
+    Returns:
+        str: The version of the cloud_defend package, or None if the API call fails.
+
+    """
+    return get_package_version(cfg=cfg, package_name="cloud_defend", prerelease=prerelease)
```

Comment:
We utilize a shared function, `get_package_version`, which accepts the `package_name` parameter. The functions `get_cloud_security_posture_version` and `get_cloud_defend_version` serve as wrapper functions

Reply:
why do we need a wrapper?

Reply:
Wrapper functions like `get_cloud_security_posture_version` streamline repeated calls across scripts. This reduces arguments by utilizing a single function that internally determines the `package_name`. Though less necessary now, these wrappers facilitate potential future logic additions. Currently, using get_package_version directly with the package name suffices. Refactoring will be less in case of wrappers.

Reply:
> Wrapper functions like get_cloud_security_posture_version streamline repeated calls across script

I know. There was only one call per wrapper, hence my question

> Though less necessary now, these wrappers facilitate potential future logic additions

In general, It's preferable to keep it simple. We can always refactor it later

Reply:
updated